test/compiler.js

```diff
@@ -583,7 +587,6 @@ function runTests (solc, versionText) {
 
       t.test('compiling standard JSON (multiple sources on old compiler)', function (st) {
         var output = JSON.parse(solc.compile('{"language":"Solidity","sources":{"cont.sol":{"content":"import \\"lib.sol\\";"},"lib.sol":{"content":""}}}'));
-        console.log(output);
```

Comment:
@axic I'm not sure why this was there, was it intended?

Reply:
No, it was typo during merging.